### PR TITLE
Elasticsearch health check on CI

### PR DESCRIPTION
**Pull request by Lutacon**
Add Elastic health check to execute tests when Elastic is ready.

### CODE REVIEW

File: etc/ci/wait-for-elastic.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+until [ "$health" = 'green' ]; do
+    health="$(curl -fsSL "$host/_cat/health?h=status")"
+    health="$(echo "$health" | sed -r 's/^[[:space:]]+|[[:space:]]+$//g')" # trim whitespace (otherwise we'll have "green ")
+    >&2 echo "Elasticsearch is unavailable - sleeping"
+    sleep 1
+done
+
+>&2 echo "Elasticsearch is up"
+exec $cmd
```
Comment (jorge07) [inline etc/ci/wait-for-elastic.sh]: You're assuming that curl will be installed. I really prefer use a container for this.

File: symfony.lock [hunk 2]
```diff
@@ -485,9 +485,6 @@
             "ref": "a249484db698d1a847a30291c8f732414ac47e25"
         }
     },
-    "symfony/security": {
```
Comment (jorge07) [inline symfony.lock]: ??

File: etc/ci/wait-for-elastic.sh (hunk 1 above)
Comment (Lutacon) [reply etc/ci/wait-for-elastic.sh]: A container to perform the health check?

Comment (jorge07) [reply etc/ci/wait-for-elastic.sh]: You can mount this into the elastic or PHP container for example. So we know we can reproduce the environment everywhere

File: makefile [hunk 3]
```diff
@@ -75,6 +75,10 @@ sh: ## gets inside a container, use 's' variable to select a service. make s=php
 logs: ## look for 's' service logs, make s=php logs
 		docker-compose logs -f $(s)
 
+.PHONY: wait-for-elastic
+wait-for-elastic: ## Health check for elastic
+		./etc/ci/wait-for-elastic.sh localhost:9200
```
Comment (jorge07) [inline makefile]: this needs to run inside a contaner

**Pull request closed without merging**